components/org.wso2.carbon.identity.captcha/src/main/java/org/wso2/carbon/identity/captcha/util/CaptchaUtil.java

```diff
@@ -380,6 +381,8 @@ private static void verifyReCaptchaEnterpriseResponse(HttpEntity entity)
                     }
                     if (score < scoreThreshold) {
                         throw new CaptchaClientException("reCaptcha score is less than the threshold.");
+                    } else if (score < warnScoreThreshold) {
+                        log.warn("User access with low reCaptcha score.");
```

Comment:
Do we need this WARN log?

Reply:
Yes. This warn log is used to trigger an alert to capture users with low recaptcha scores.